docs/source/markdown/podman-stats.1.md

```diff
@@ -32,15 +32,32 @@ Valid placeholders for the Go template are listed below:
 
 | **Placeholder** | **Description**    |
 | --------------- | ------------------ |
-| .ID             | Container ID       |
+| .ID             | Container ID (truncated) |
+| .ContainerID    | Container ID (full hash) |
 | .Name           | Container Name     |
 | .CPUPerc        | CPU percentage     |
 | .MemUsage       | Memory usage       |
 | .MemUsageBytes  | Memory usage (IEC) |
+| .AvgCPU         | Average CPU (full precision float) |
+| .AVGCPU         | Average CPU (formatted) |
+| .CPU            | Percent CPU (full precision float) |
+| .CPUNano        | CPU Usage, total, in nanoseconds |
+| .CPUSystemNano  | CPU Usage, kernel, in nanoseconds |
+| .Duration       | Same as CPUNano (FIXME: why?) |
+| .MemLimit       | Memory limit, in bytes |
 | .MemPerc        | Memory percentage  |
 | .NetIO          | Network IO         |
 | .BlockIO        | Block IO           |
 | .PIDS           | Number of PIDs     |
+| .NetInput       | Network Input      |
+| .NetOutput      | Network Output     |
+| .BlockInput     | Block Input        |
+| .BlockOutput    | Block Output       |
+| .PerCPU         | FIXME: empty array? |
+| .PIDs           | Number of PIDs     |
+| .SystemNano     | Current system datetime, nanoseconds since epoch |
+| .UpTime         | WTF? This is Duration (CPUNano) in human-readable form?? |
+| .Up             | Same as UpTime (FIXME: why?) |
```

Comment:
The rest of the man pages can be slowly filled in if/when people like, but this set here bothers me a little: some of the fields are duplicated for no clear reason; some (Duration?) seem misleading to my untrained eye. I wonder if someone familiar with `podman stats` would mind taking a look and confirming that these fields all do what's desired?